frontends/web/src/api/account.ts

```diff
@@ -45,9 +45,9 @@ export const init = (code: string): Promise<null> => {
     return apiPost(`account/${code}/init`);
 };
 
-export const reinitialize = (): Promise<null> => {
-    return apiPost('accounts/reinitialize');
-};
+// export const reinitialize = (): Promise<null> => {
```

Comment:
I slightly prefer to not have commented or unused code in master, so I'd just delete it and add it in the commit where it is gonna be used.

Reply:
done https://github.com/digitalbitbox/bitbox-wallet-app/compare/78f50875e799d2914383d768b30ba632454a0fab..ca06edf99cf5be6a0e9a7b913f3f9ffc2e0e0bba